Libs/MRML/Widgets/qMRMLVolumeInfoWidget.cxx

```diff
@@ -368,12 +369,22 @@ bool qMRMLVolumeInfoWidget::isCentered()const
 void qMRMLVolumeInfoWidget::center()
 {
   Q_D(qMRMLVolumeInfoWidget);
-  double origin[3];
-  if (!d->centeredOrigin(origin))
+  double centerOrigin[3];
+  if (!d->centeredOrigin(centerOrigin))
     {
     return;
     }
-  d->VolumeNode->SetOrigin(origin);
+  double volumeOrigin[3];
+  d->VolumeNode->GetOrigin(volumeOrigin);
+  vtkNew<vtkMatrix4x4> transformMatrix;
+  vtkNew<vtkMRMLLinearTransformNode> centeringTransform;
+  transformMatrix->SetElement(0, 3, centerOrigin[0]-volumeOrigin[0]);
+  transformMatrix->SetElement(1, 3, centerOrigin[1]-volumeOrigin[1]);
+  transformMatrix->SetElement(2, 3, centerOrigin[2]-volumeOrigin[2]);
+  centeringTransform->SetName("CenteringTransform");
```

Comment:
I suggest to name the transform by including the volume name.

instead of `CenteringTransform` it would be `<volumeNodeName>-CenteringTransform` (e.g `CTChest-CenteringTransform`)

Reply:
That's a good point, it improves the clarity since no volume node would have the same name as the others'. I've made the changes. 
![renameTransform](https://user-images.githubusercontent.com/8678893/88117535-685b6680-cb89-11ea-913d-cc565a43e8e5.png)